examples/gke-security-best-practices/builder/platform-uploads/tmp/acme-cas.pub

```diff
@@ -0,0 +1,3 @@
+ssh-rsa {base64 goes here}
```

Comment:
From where? Confused how this is meant to work.

Reply:
this is not required, I will remove this step.

Reply:
I have removed the packer scripts in favor of a GCP Marketplace CIS security hardened image for the bastion host.